config/locales/two_factor_authentication/en.yml

```diff
@@ -187,8 +187,8 @@ en:
       additional_methods_link: choose another authentication method
       error_page_text: You have face or touch unlock enabled for this account. Use the
         same device and browser profile each time.
-      multiple_methods: Face or touch unlock was unsuccessful. Please try again or %{link}.
       title: We can’t identify your device
+      try_again: Face or touch unlock was unsuccessful. Please try again or %{link}.
```

Comment:
The idea with the rename is that, as of LG-8622 / #8716, we no longer differentiate the error behavior based on whether the user has multiple MFA methods enabled.